submissions/korotushak96/a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,84 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+
+// ======== OUTPUT ========
+/* Use print(message) for output.
+   Default tag for message is <pre>. Use print(message,'div') to change containing element tag.
+
+   Message can contain HTML markup. You may also tweak index.html and/or styles.css.
+   However, please, REFRAIN from improving visuals at least until your code is reviewed
+   so code reviewers might focus on a single file that is index.js.
+   */
+
+/* Print examples:
+   print('ABC');
+   print('<strong>ABC</strong>');
+   print('<strong>ABC</strong>', 'div');
+
+   print('human; John; male; 2; 2; Hello world!; Rex, Tom, Jenny');
+   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny');
+   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny', 'div');
+   */
+const dog = {
+    species: 'dog',
+    name: 'Dina',
+    gender: 'female',
```

Comment:
Try define constants for repeated properties

Reply:
I add classes. But I'm not sure I used correct inheritance. Please, check it

Reply:
@korotushak96 You don't need use ES6 Class here. Please, return to simple `Object` definition. You will improve this task with ES6 Class in future.

Reply:
IMO looks like this.
```javascript
const INHABITANT = {
    species: {
        dog: 'dog',
        human: 'human',
        ...etc
    },
    genders: {
        male: 'male',
        female: 'female',
    }
};
const dog = {
    name: "Paolo",
    species: INHABITANT.species.dog,
    gender: INHABITANT.genders.male,
    legs: 4,
    hands: 0,
    saying: "Bark",
};
...etc
```

Reply:
ok =) done